src/pudl/output/helpers.py

```diff
@@ -0,0 +1,33 @@
+"""Helper functions for creating output assets."""
+import importlib
+
+from dagster import AssetsDefinition, asset
+
+
+def sql_asset_factory(
```

Comment:
There might be a better location for this function.

Reply:
This does seem like a weird place for it. How about `pudl.output.sql` or `pudl.output.sql_views`?

Do we want to think about how the assets should ultimately be organized and start populating that hierarchy with the new assets that we're creating? Something based on Dagster's recommended project layout? What would the sub-packages under `pudl.assets` be?

It seems like the layout becomes much more flexible once we're using assets and referring to them in the global asset namespace, rather than calling functions and passing all the dataframes around in function calls.